### PR TITLE
[s]VV now escapes name (because name could be an item in a list)

### LINKED ISSUE #19011: Poly managed to bust VV with a dodgy phrase
Someone taught poly this phrase:

`Ruel*/*;>gerh**&igned<%u *; St**o */*neer `

And in the VV window it rendered as:

`Ruel*/*;>gerh**&igned`

At which point the window terminated without printing any more vars.

Removing that phrase from the list fixed VV, but still.

### PR DESCRIPTION
**Pull request by MrsTonedOne**
fixes #19011

Security tag because somebody could abuse this somehow to make vv play a youtube embed on open or even XSS/XSRF to trigger admin actions.

(name meaning the name of the "variable")

### CODE REVIEW

File: code/datums/datumvars.dm [hunk 1]
```diff
@@ -414,7 +414,7 @@ body
 				html += "</ul>"
```
Comment (MrStonedOne) [inline code/datums/datumvars.dm]: You can't see it without expanding the diff, but a few lines above this shows how viewing a list's contents recursively calls this same proc.

**Comment by optimumtact** [comment]
@AnturK that tag is for ingame security, not this security :laughing:

**Comment by lzimann** [comment]
this broke attack log formatting with colors =/

**Comment by phil235** [comment]
Breaks the color coding of the attack_log text.

**Pull request merged**